crates/stc_ts_types/src/name.rs

```diff
@@ -20,6 +20,14 @@ type Inner = SmallVec<[Id; 4]>;
 pub struct Name(Inner);
 
 impl Name {
+    pub fn new(ctxt: SyntaxContext, name: JsWord) -> Self {
```

Comment:
Please change order

Reply:
Are there criteria in the order of factor values?